pkg/cli/sql_util.go

```diff
@@ -989,23 +999,33 @@ func maybeShowTimes(
 	}
 
 	// If discrete server/network timings are available, also print them.
-	execLatency, serviceLatency, err := conn.getLastQueryStatistics()
+	parseLat, planLat, execLat, serviceLat, err := conn.getLastQueryStatistics()
 	if err != nil {
 		fmt.Fprintf(stderr, "\nwarning: %v", err)
 		return
 	}
 
-	networkLatency := clientSideQueryTime - serviceLatency
+	fmt.Fprint(stderr, " total")
+
+	networkLat := clientSideQueryTime - serviceLat
+	otherLat := serviceLat - parseLat - planLat - execLat
 	if sqlCtx.verboseTimings {
-		fmt.Fprintf(w, " total (exec %s / net %s / other %s)\n",
-			execLatency, networkLatency, serviceLatency-execLatency)
-	} else {
-		// Simplified display: just show percentages.
-		totalSeconds := clientSideQueryTime.Seconds()
-		fmt.Fprintf(w, " total (exec %.1f%% / net %.1f%% / other %.1f%%)\n",
-			execLatency.Seconds()*100/totalSeconds,
-			networkLatency.Seconds()*100/totalSeconds,
-			(serviceLatency-execLatency).Seconds()*100/totalSeconds)
+		fmt.Fprintf(w, " (parse %s / plan %s / exec %s / other %s / network %s)\n",
```

Comment:
would you consider it in-scope for this PR to always include the verbose timings if in EXPLAIN ANALYZE?

Reply:
huh not really no

Reply:
fine by me. it's something to consider for later based on the feedback from customer-support. (or maybe we should just make sure the CS team knows about `\set verbose_timings`)

Reply:
the issue is that this would require client-side parsing which is not any more part of the "presentation" logic.